src/datumaro/plugins/yolo_format/extractor.py

```diff
@@ -472,39 +472,59 @@ def _get_subset_image_paths(self, subset_name: str):
         else:
```

Comment:
I think it could be nice to also report the annotation (or line) number in https://github.com/cvat-ai/datumaro/blob/1175180cd0a633e39d2af0293dcce63882c83b07/src/datumaro/plugins/yolo_format/extractor.py#L287-L293